billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/series/webservice/Client.java

```diff
@@ -0,0 +1,311 @@
+            }
+        }
+        return null;
+    }
+
+    private String convert(SeriesProcessingMedium processingMedium){
+        if (processingMedium != null){
+            switch (processingMedium){
+                case COMPUTER_PROGRAM: return "PI";
+                case FINANCAS_WEBPORTAL: return "PF";
+                case OTHER_ELECTRONIC: return "OM";
+                default:
+                    throw new RuntimeException("unknown processing medium: " + processingMedium);
+            }
+        }
+        return null;
+    }
+
+    private ResultInfo convert(OperationResultInfo resultInfo, int successCode) {
+        if (resultInfo != null) {
+            return new ResultInfo(
+                resultInfo.getCodResultOper().intValue(),
+                resultInfo.getMsgResultOper(),
+                resultInfo.getCodResultOper().intValue() == successCode);
+        }
+        return null;
+    }
+
+    private SeriesInfo convert(com.premiumminds.billy.portugal.webservices.series.SeriesInfo seriesInfos){
+        if (seriesInfos != null){
+            final SeriesInfo rvalue = new SeriesInfo();
+            rvalue.setSerie(seriesInfos.getSerie());
+            rvalue.setTipoSerie(seriesInfos.getTipoSerie());
+            rvalue.setClasseDoc(seriesInfos.getClasseDoc());
+            rvalue.setTipoDoc(seriesInfos.getTipoDoc());
+            rvalue.setNumInicialSeq(seriesInfos.getNumInicialSeq());
+            rvalue.setNumFinalSeq(seriesInfos.getNumFinalSeq());
+            rvalue.setDataInicioPrevUtiliz(convertLocalDate(seriesInfos.getDataInicioPrevUtiliz()));
+            rvalue.setSeqUltimoDocEmitido(seriesInfos.getSeqUltimoDocEmitido());
+            rvalue.setMeioProcessamento(seriesInfos.getMeioProcessamento());
+            rvalue.setNumCertSWFatur(seriesInfos.getNumCertSWFatur());
+            rvalue.setCodValidacaoSerie(seriesInfos.getCodValidacaoSerie());
+            rvalue.setDataRegisto(convertLocalDate(seriesInfos.getDataRegisto()));
+            rvalue.setEstado(seriesInfos.getEstado());
+            rvalue.setMotivoEstado(seriesInfos.getMotivoEstado());
+            rvalue.setJustificacao(seriesInfos.getJustificacao());
+            rvalue.setDataEstado(convertLocalDateTime(seriesInfos.getDataEstado()));
+            rvalue.setNifComunicou(seriesInfos.getNifComunicou());
+            return rvalue;
+        }
+        return null;
+    }
+
+    private LocalDateTime convertLocalDateTime(XMLGregorianCalendar date) {
+        if (date != null) {
+            return date.toGregorianCalendar().toZonedDateTime().toLocalDateTime();
+        }
+        return null;
+    }
+
+    private LocalDate convertLocalDate(XMLGregorianCalendar date) {
+        if (date != null) {
+            return date.toGregorianCalendar().toZonedDateTime().toLocalDate();
+        }
+        return null;
+    }
+
+    private XMLGregorianCalendar convertLocalDate(final LocalDate date) {
```

Comment:
This method and the previous having the same name looks a bit strange. Think you can change at least one of them to avoid confusion?